Release.md

```diff
@@ -0,0 +1,104 @@
+# FuseML Release Process
+
+## Repositories under release management 
+
+Every repository that is part of release management features one release branch
+for every official FuseML release. Currently, release branches are created to track
+every minor release, e.g. `v0.1` and `v0.2`, but this may change after FuseML exits alpha
+status.
+
+The next sections document the release artifacts for every repository that is part
+of release management, as well as the code changes that need to be applied to the
+release branch whenever a new release is created and the release trigger.
+
+### fuseml/fuseml
+
+#### Release artifacts
+
+* fuseml-installer binary
+
+#### Release changes
+
+* update the fuseml-core k8s deployment manifest to use the release fuseml-core image tag instead of the `dev` tag
+* update the default extension repository to point to the `release-*` extension branch instead of `main`
+
+Example: https://github.com/fuseml/fuseml/pull/211
+
+#### Release triggers
+
+* push a new tag in the release branch; the rest of the release is automated through github actions
+
+### fuseml/fuseml-core
+
+#### Release artifacts
+
+* fuseml CLI binary
+* fuseml-core binary
+* fuseml-core container image
+
+#### Release changes
+
+Nothing needs to be done. Everything is automated through github actions and the Makefile
+
+#### Release triggers
+
+* push a new tag in the release branch; the rest of the release is automated through github actions
+
+### fuseml/extensions
+
+#### Release artifacts
+
+* extension container images
+  * mlflow-builder
+  * kfserving-predictor
+* mlflow tracker container image
+* extension installer descriptor files
+
+#### Release changes
+
+None
+
+#### Release triggers
+
+* manual release process; container images must be built and published manually
+
+### fuseml/examples
+
+#### Release Artifacts
+
+* example codesets
+* example workflows
+* example webapps
+
+#### Release changes
+
+* update the workflow examples to point to the released container image tags. Example: https://github.com/fuseml/examples/pull/11
+* build/release container images for webapp examples
```

Comment:
I wonder if its really needed to build a new image for the webapp. AFAIK it does not include anything related to fuseml and the only resource that references it is the knative service that is also part of the repository.

Reply:
Okay, I'll mark it as optional.